demo/src/main/AndroidManifest.xml

```diff
@@ -1,7 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
-    xmlns:tools="http://schemas.android.com/tools"
-    package="com.google.maps.android.utils.demo">
```

Comment:
Would you comment why this attribute should be removed? Seems like a best practice changed to put the namespace in the build.gradle file?

Reply:
Hi @wangela. This has been deprecated in AGP 7.3.0, see official statement [here](https://developer.android.com/studio/releases/gradle-plugin#package-deprecated). The package should now be specified as a namespace in the Gradle file.